### PR TITLE
Add colored circle test glyphs for v0 and v1

### PR DESCRIPTION
**Pull request by drott**
Preparation for testing custom palette functionality. Introduce two
colored circle glyphs, one as COLRv0, one as COLRv1, going through the
colors of the rainbow from the outside to the inside.

Update logic inside test font generator script to allow adding COLRv0
glyphs to the COLR table, using COLRv0 style layer tuples as arguments
to buildCOLR().

Mark the new glyphs with an extra layer that shows a 0 or a 1 to
indicate the COLR glyph format (i.e. layers in v0, a graph of Paint
structures in v1).

### CODE REVIEW

**Comment by drott** [comment]
image

File: config/more_samples-glyf_colr_1.py [hunk 1]
```diff
@@ -219,7 +223,7 @@ def _gradient_p2_skewed(accessor_char):
     return SampleGlyph(
         glyph_name=glyph_name,
         accessor=accessor_char,
-        advance=_UPEM,
+        advance=1250,
```
Comment (rsheeter) [inline config/more_samples-glyf_colr_1.py:226]: Suggest adding a _ADVANCE const

Comment (rsheeter) [reply config/more_samples-glyf_colr_1.py:226]: ...in reading further I see it's not consistent. Maybe just a comment ot explain why it isn't _UPEM?

File: config/more_samples-glyf_colr_1.py [hunk 2]
```diff
@@ -802,6 +806,161 @@ def _foreground_color(fill_type, foreground_alpha, accessor_char):
...
+            circle_reference(350, "red"),
+            circle_reference(300, "orange"),
+            circle_reference(250, "yellow"),
+            circle_reference(200, "green"),
+            circle_reference(150, "blue"),
+            circle_reference(100, "indigo"),
+            circle_reference(50, "violet"),
+            {
+                "Format": ot.PaintFormat.PaintGlyph,
+                "Glyph": f"one",
```
Comment (rsheeter) [inline config/more_samples-glyf_colr_1.py:860]: nit: unnecessary f; nothing is interpolated in

**Review by rsheeter — APPROVED** [review]

**Pull request merged**